alsa-lib/tests/test.bats

```diff
@@ -0,0 +1,18 @@
+@test "package directory for package ident ${TEST_PKG_IDENT} exists" {
```

Comment:
I'm not sure this test makes sense in this context. We shouldn't have to re-test hab functionality in a core plan.

Reply:
I'd prefer to keep this test, but I'll remove this and keep the discussion in https://github.com/habitat-sh/core-plans/pull/2604